src/utils/components/PolicyForm/PolicyInterface.tsx

```diff
@@ -185,14 +193,53 @@ const PolicyInterface: FC<PolicyInterfaceProps> = ({
           isChecked={!!policyInterface.ipv4}
           onChange={onIP4Change}
         />
-        {!!policyInterface.ipv4 && (
-          <Checkbox
-            label={t('DHCP')}
-            id={`policy-interface-dhcp-${id}`}
-            isChecked={policyInterface.ipv4.dhcp}
-            onChange={onDHCPChange}
-          />
-        )}
+        <div className="pf-u-ml-md pf-u-mt-sm">
+          {!!policyInterface.ipv4 && (
+            <Checkbox
+              label={t('DHCP')}
+              id={`policy-interface-dhcp-${id}`}
+              isChecked={policyInterface.ipv4.dhcp}
+              onChange={onDHCPChange}
+            />
+          )}
+
+          {!!policyInterface.ipv4?.dhcp && (
+            <Checkbox
+              label={t('Auto-DNS')}
+              id={`policy-interface-dns-${id}`}
+              isChecked={policyInterface.ipv4['auto-dns']}
+              onChange={(checked) =>
+                onInterfaceChange((draftInterface) => (draftInterface.ipv4['auto-dns'] = checked))
+              }
+            />
+          )}
+
+          {!!policyInterface.ipv4?.dhcp && (
+            <Checkbox
+              label={t('Auto-routes')}
+              id={`policy-interface-routes-${id}`}
+              isChecked={policyInterface.ipv4['auto-routes']}
+              onChange={(checked) =>
+                onInterfaceChange(
+                  (draftInterface) => (draftInterface.ipv4['auto-routes'] = checked),
+                )
+              }
+            />
+          )}
+
+          {!!policyInterface.ipv4?.dhcp && (
```

Comment:
We are repeating `!!policyInterface.ipv4?.dhcp &&...` here three times in a row.. Maybe we could do it nicer, maybe to check the condition once and then to display all the checkboxes according to the check? WDYT?